test/jobs/reset.js

```diff
@@ -66,13 +66,8 @@ test('reset repo', async t => {
 
   t.afterEach(async () => {
     nock.cleanAll()
-    await removeIfExists(repositories, '42')
-    await installations.remove(await installations.get('123'))
-    await removeIfExists(repositories, '42:pr:123')
-    await removeIfExists(repositories, '42:branch:deadbeef')
-    await removeIfExists(repositories, '42:branch:deadbeef0')
-    await removeIfExists(repositories, '42:issue:67')
-    await removeIfExists(repositories, '42:issue:65')
+    await removeIfExists(repositories, '42', '42:pr:123', '42:branch:deadbeef', '42:branch:deadbeef0', '42:issue:67', '42:issue:65')
```

Comment:
:art: line gets a little long, no biggie overall, but we might wanna think about a default format with more line breaks, or passing an array that we construct over multiple lines.

Reply:
In bffa40b I refactored `removeIfExists` to also take an array, in 899a57e I'm using it to make the line more readable :)